components/events.tsx

```diff
@@ -74,6 +136,7 @@ const Events = () => {
 			});
 		}
 		setNfcId('');
+		getData();
```

Comment:
calling getData() so that when a user is checked in it performs a GET request & re-renders the event count